controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/RoleDefinition.java

```diff
@@ -20,60 +20,46 @@
     /** Deus ex machina. */
     hostedOperator(Policy.operator),
 
+    /** Reader - the base role for all tenant users */
+    publicReader(
+            Policy.tenantRead,
+            Policy.applicationRead,
+            Policy.deploymentRead,
+            Policy.publicRead
+    ),
+
+    /** User - the dev.ops. role for normal Vespa tenant users */
+    publicDeveloper(
+            Policy.applicationCreate,
+            Policy.applicationUpdate,
+            Policy.applicationDelete,
+            Policy.developmentDeployment
+    ),
+
+    /** Admin - the administrative function for user management etc. */
+    publicAdministrator(
+            Policy.tenantUpdate,
+            Policy.tenantManager
+    ),
+
+    /** Headless - the application specific role identified by deployment keys for production */
+    publicHeadless(
+            Policy.submission,
+            Policy.deploymentPipeline,
+            Policy.productionDeployment
+    ),
+
```

Comment:
A role which a `Developer` assumes when authenticating through a signed request is missing. I believe we wanted to include only the "read" and "development deployment" policies in that.

Reply:
Nice catch. `developerWithKey` or something like that?

Reply:
I suppose.